### ISSUE TITLE
We need to recompile android for each games ?

### ISSUE THREAD

**Issue by takhimi** [jlopezcur/GodotGooglePlayGameServices#14]
I just want to clear few things on my mind. Since we need to change the APP_ID to our game's specific ID, if I have 20 games with different ID , I need to recompile 20 times to get 20 different APK ?

Am I correct ? Can this gpgs module still can be use ? Last updated was 2017

**Comment by ANOONBP** [comment]
Yep you need to recompile for each games and yep this module still can be use but have to modify some code to avoid gradlew build failed.

**Comment by anisc** [comment]
@ANOONBP could you plz share the modification that need to be made ?
